ingest.go

```diff
@@ -522,19 +523,21 @@ func ingestTargetLevel(
 		iter := newLevelIter(iterOps, cmp, nil /* split */, newIters,
 			v.L0Sublevels.Levels[subLevel].Iter(), manifest.Level(0), nil)
 
-		var rangeDelIter keyspan.FragmentIterator
-		// Pass in a non-nil pointer to rangeDelIter so that levelIter.findFileGE
-		// sets it up for the target file.
-		iter.initRangeDel(&rangeDelIter)
+		rangeDelIter := keyspan.LevelIter{}
+		rangeDelIter.Init(keyspan.SpanIterOptions{RangeKeyFilters: iterOps.RangeKeyFilters}, cmp,
+			tableNewRangeDelIter(context.Background(), newIters), v.L0Sublevels.Levels[subLevel].Iter(),
+			manifest.Level(0), manifest.KeyTypePoint)
+		fragmentRangeDelIter := keyspan.FragmentIterator(&rangeDelIter)
 
 		levelIter := keyspan.LevelIter{}
```

Comment:
nit: since there are two leveliters (three if you count the point levelIter), I'd rather call this `rangeKeyIter` to keep it obvious.